docs/providers/aws/guide/serverless.yml.md

```diff
@@ -365,6 +368,12 @@ functions:
           maximumRecordAgeInSeconds: 120
           startingPosition: LATEST
           enabled: false
+      - msk:
+          arn: arn:aws:kafka:us-east-1:111111111111:cluster/ClusterName/a1a1a1a1a1a1a1a1a
+          topic: kafkaTopic
+          batchSize: 100
+          startingPosition: LATEST
+          enabled: false
```

Comment:
When there are defaults, let's use _default_ values (so `true` in this case). Also it's nice to note in comment on optional properties

Reply:
Thanks for catching that - I followed the same convention that I saw in `stream` event right above it - should it also have `enabled` set to `true` here?

Reply:
> should it also have enabled set to `true` here?

Yes, definitely, somehow it was missed